### PR TITLE
add steps on how to update the submodules in the readme

### PR DESCRIPTION
**Pull request by vpchung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,19 @@ TBA
 
 ### Running with Docker
 
-1. git clone --recursive https://github.com/Sage-Bionetworks/rocc-app.git
+1. Clone this repository, as well as the two linked repositories.  This is done
+with the `--recursive` flag.
+
+       git clone --recursive https://github.com/Sage-Bionetworks/rocc-app.git
+
+Refer to **Updating a Linked Repository** section below if you need to update one
```
Comment (tschaffter) [inline README.md:22]: Use an anchor that allows the user to jump to this section.

Comment (tschaffter) [reply README.md:22]: For a future update

**Review by tschaffter — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vpchung) [reply README.md]: Thanks for the suggestion.

Done :white_check_mark:

**Comment by tschaffter** [comment]
@vpchung Please prefix inline lines of code with 4 spaces (currently 7). For code longer than one line, we can follow [Google's recommendation](https://google.github.io/styleguide/docguide/style.html#code) and use codeblocks with the language specified.

**Pull request merged**